api/explorer/api/test/test_ingest.py

```diff
@@ -41,7 +41,7 @@
     build_inputs=[
         model.BuildInput(
             build_input_type=model.BuildInputType.BUILD_INPUT,
-            package=dummy_pkg_c,
+            output_path="/C",
```

Comment:
why do you use a literal here?

Reply:
Agreed. I think we should be able to get the output path from `dummy_pkg_c.output_paths` which is defined above, right?